pkg/agent/openflow/pipeline.go

```diff
@@ -1777,10 +1736,10 @@ func (f *featureNetworkPolicy) conjunctionActionFlow(conjunctionID uint32, table
 			}
 			if l7RuleVlanID != nil {
 				return fb.
-					Action().LoadToRegField(conjReg, conjunctionID).                                                // Traceflow.
-					Action().LoadRegMark(DispositionAllowRegMark, CustomReasonLoggingRegMark, L7NPRedirectRegMark). // AntreaPolicy, Enable logging.
-					Action().SendToController(uint8(PacketInReasonNP)).
-					Action().CT(true, nextTable, ctZone, f.ctZoneSrcField). // CT action requires commit flag if actions other than NAT without arguments are specified.
+					Action().LoadToRegField(conjReg, conjunctionID).                                                 // Traceflow.
+					Action().LoadRegMark(DispositionAllowRegMark, L7NPRedirectRegMark).                              // AntreaPolicy.
+					Action().SendToController([]byte{uint8(PacketInCategoryNP), PacketInNPLoggingOperation}, false). // Enable logging.
```

Comment:
This will lead to two packetin calls or a single packetin with merged categories?

Reply:
Sorry, I'm not sure if I understand your question. L1741 will send this packet to the controller and only NP packetIn handler will be called once to operate logging.

Reply:
I misunderstood the byte slice. Now I see the first byte is category and the second one is another kind of data.